src/components/SearchableDropdown/SearchableDropdown.stories.tsx

```diff
@@ -2,7 +2,7 @@ import { useState } from "react";
 import SearchableDropdown from ".";
 
 export default {
-  title: "Searchable Dropdown",
+  title: "Components/Searchable Dropdown",
   component: SearchableDropdown,
 };
 
```

Comment:
nit. Could you change the `Story` to `Default` in this one too?

<img width="224" alt="Screen Shot 2022-07-01 at 12 02 31 PM" src="https://user-images.githubusercontent.com/47064971/176930303-22b0b465-7408-4586-81d2-504959e1bb43.png">